workload_version

```diff
@@ -0,0 +1 @@
+6.0.6
```

Comment:
might be nice if we have an integration test that verifies this file is up to date

Reply:
Yes!
I'll add such tests on all charms.